flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/SqlClientTest.java

```diff
@@ -60,8 +60,6 @@
 
```

Comment:
Could you add `@Rule public Timeout timeout = new Timeout(10, TimeUnit.SECONDS);` in this test class? This can avoid the test hangs.